Code/BasicFilters/json/RegionOfInterestImageFilter.json

```diff
@@ -12,7 +12,7 @@
       "default" : "std::vector<unsigned int>(3, 1)",
       "dim_vec" : 1,
       "itk_type" : "typename InputImageType::SizeType",
-      "briefdescriptionSet" : "",
+      "briefdescriptionSet" : "Size in pixels of the region extracted.",
```

Comment:
I think it should read "extracted region." and not "region extracted."

Reply:
This comes from ITK